README.md

```diff
@@ -76,18 +116,207 @@ load(
 )
 ```
 
-Then use `codechecker_test()` rule passing targets you call CodeChecker for:
+Create a `codechecker_test()` target by passing other targets you'd like CodeChecker to analyze:
+
+<!-- TODO: Consider using https://github.com/bazelbuild/stardoc to document parameters -->
+```python
+codechecker_test(
+    name = "your_codechecker_rule_name",
+    targets = [
+        "your_target",
+    ],
+)
+```
+
+Then invoke bazel:
+
+```bash
+bazel build ://your_codechecker_rule_name
+```
+
+You can find the analysis results in the `bazel-bin/` folder, on which you
+can run [`CodeChecker store`](https://github.com/Ericsson/codechecker/blob/master/docs/web/user_guide.md#store)
+or [`CodeChecker parse`](https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/user_guide.md#parse):
+
+```bash
+CodeChecker parse bazel-bin/path/to/results
+```
+
+### Build-only CodeChecker analysis: `codechecker()`
+
+This rule is functionally equivalent to `codechecker_test` but omits the test phase where either PASS or FAIL is printed.
+You can include and use it similarly as well:
+
+```python
+load(
+    "@bazel_codechecker//src:codechecker.bzl",
+    "codechecker"
+)
+```
```

Comment:
It is good as explanation, but I dont think we have any use-cases for codechecker() rule. For the user everything can be achieved via codechecker_test(). I actually consider it as internal rule.